### DISCUSSION TITLE
Add pixi as an environment manager option

### DISCUSSION THREAD

**Discussion by cnrrobertson** [drivendataorg/cookiecutter-data-science#406] [Requested tools]
I have been migrating all of my data science work from `conda` to [`pixi`](https://github.com/prefix-dev/pixi/) over the six months. It has benefits in speed and robustness for resolving and installing dependencies from either `conda-forge` or `pip` and makes use of a much more deterministic, cross platform, and clean manifest (like poetry). Would there be any interest in adding this as a choice for environment manager?

One note is that `pixi` has its own dependency file format `pixi.toml` but can also make use of `pyproject.toml` files as a manifest (see [here](https://pixi.sh/latest/advanced/pyproject_toml/#dependency-section)) which could build on the current template structure.

If there is interest, I can put together a PR.

**Comment by jamesmyatt** [comment]
I've been using pixi lately and it's been great.

**Comment by pjbull** [comment]
Released in v2.3.0. Upgrade and use the latest version to check it out: https://pypi.org/project/cookiecutter-data-science/
